site/slices/new-contact-us-slice/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import classnames from 'classnames/bind';
+import styles from './style.css';
+
+const cx = classnames.bind(styles);
+
+class NewContactUs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isHovered: false,
+    };
+  }
+
+  onHover = () => {
+    this.setState({
+      isHovered: true,
+    });
+  };
+
+  onBlur = () => {
```

Comment:
Is `onBlur` the appropriate name here? `offHover` maybe? hmm I'm not good with the names today... 😄

Reply:
@davidbasalla The onBlur event is native to HTML